src/index.html

```diff
@@ -11,7 +11,173 @@
+           required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+           type="text"
+           name="name"
+           autocomplete="off"
+           required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+           type="number"
+           name="age"
+           min="1"
+           max="100"
+           value="12"
+           required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+           type="date"
+           name="dateOfBirth"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+           type="checkbox"
+           name="terms"
+           required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+           type="email"
+           name="email"
+           data-qa="field-placeholder"
+           placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+           type="password"
+           name="password"
+           minlength="7"
+           maxlength="14"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+
+          <input
+           type="radio"
+           name="loveCats"
+          >
+          Yes
+
+          <input
+           type="radio"
+           name="dontLoveCats"
+          >
+          No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+           type="color"
+           name="favColor"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+           type="time"
+           name="sleepTime"
+          >
+        </label>
+
+        <label class="form-field">
+         What are your favorite brand of cars?
+          <select
+           name="cars"
+           multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input
+           type="range"
+           name="rating"
+           min="1"
+           max="10"
+           value="1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="form-field" autocomplete="off">
+          Comments:
+          <textarea
+           name="info"
+           rows="2"
+           cols="20">
+          </textarea>
```

Comment:
```suggestion
          <textarea
           name="info"
           rows="2"
           cols="20"
          ></textarea>
```

Reply:
didn't fix it

Reply:
unfortunately my lint doesn´t accept this type of formatting and I am getting errors